vtex-apps/my-account
# Magiclink feature 🧙 on 1.x

#### What did you change? \*

- Return to the origin url passing the newly created addressId as argument.

#### Why? \*

- Enable the MagicLink feature.

#### How to test it? \*

- [Login](https://felipe--recorrenciaqa.myvtex.com) with `assinaturasqa@mailinator.com`
- [Go to](https://felipe--recorrenciaqa.myvtex.com/account#/addresses/new?returnUrl=/subscriptions/6FEA11DCB0197FAC3F46B3836EA15903)
- Create a new address
- Check if it was added to the URL through the argument `newAddressId`


Same as: https://github.com/vtex-apps/my-account/pull/162

#### Types of changes \*
